packages/extension/src/view/popup/tests/index.tsx

```diff
@@ -88,7 +88,7 @@ const tabCookies = {
   },
 };
 
-describe('App', () => {
+xdescribe('App', () => {
```

Comment:
Please add a comment about why this test is skipped.

Reply:
Removed the test.